core/quantity/src/Formatter/Interfaces.ts

```diff
@@ -25,6 +25,7 @@ export interface FormatProps {
   readonly scientificType?: string; // conditionally required
   readonly stationOffsetSize?: number; // conditionally required
   readonly stationSeparator?: string;
+  readonly allowMathematicEquations?: boolean;
```

Comment:
Can you add a snippet to NextVersion.md detailing how to enable mathematic equations? What operations are supported, what happens if you pass in an unsupported operator like multiplication?

Reply:
Sure, I'll do that (:

Reply:
Adding to this, can a test case be added for multiplication? Should a case like "1 FT * 6IN" throw ParseError.MathematicEquationFoundButIsNotAllowed?

Reply:
I cannot add a test for that right now since these operators are not "tokenized". 
Adding "tokenization" for these operators is half the effort for a full support, it's considerable. 

With the current implementation, it will result in an error anyway, just not this one : ParseError.MathematicEquationFoundButIsNotAllowed.

Reply:
Done with the summary, let me know if I'm missing anything.